pages/[id]/bridge.tsx

```diff
@@ -108,7 +123,6 @@ const Bridge = () => {
     error: bridgeToL2Error,
   } = useEasyWrite({
     enabled:
-      isNonZeroInput &&
```

Comment:
nit: why remove here? do we want this hook to run when the input is 0?

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/157/commits/283e41df1bf91e1a6eba8e6be69699ab7fa38a85